tidy3d/web/webapi.py

```diff
@@ -461,15 +481,18 @@ def _upload_task(  # pylint:disable=too-many-locals,too-many-arguments
     # with Progress() as progress:
     # upload_progress = UploadProgress(size_bytes, progress)
     log.debug(f"json = {json_string}")
-    client.put_object(
+    resp = client.put_object(
         Body=json_string,
         Bucket=bucket,
         Key=key,
         # Callback=upload_progress.report
     )
 
-    task["status"] = "draft"
-    http.put(f"{method}/{task_id}", data=task)
+    if resp.get("ResponseMetadata") and resp.get("ResponseMetadata").get("HTTPStatusCode") != 200:
+        raise WebError(f"Upload file to 3S failed, please check your network or try it later.")
+
+    # task["status"] = "draft"
```

Comment:
@momchil-flex could u check if we still need draft status from python client side?

Reply:
if it is just medium status, we can remove the code.